index.html

```diff
@@ -168,6 +168,31 @@ <h3>Skills</h3>
         
     </ul>
 </section>
-
+<section>
+    <h3 class="info3">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h3>
+    </section>
+    <section class="footer">
+    <form
+    action="https://formspree.io/f/xbjenlov"
+    method="POST">
+        <ul>
+            <li><input type="text" class="name-email" name="name" maxlength="30" placeholder="Full name" required></li>
+             <li> <input type="text" class="name-email" name="email" placeholder="Email address" required></li>
+              <li> <textarea class="message" maxlength="500" id="msg" name="user_message"  placeholder="Hey I’d like to connect to..." required></textarea></li>
+               <li><button type="submit" class="my-resume2">Get in touch</button></li>         
+       </ul>
+    </form>
+    <h3 class="info4">myemailhere@email.com</h3>
+<div class="last-img3"><img src="images/illustration-contact-form.png" alt="illustration 3"></div>
+<ul class="footer-icons">
+<li><img src="./Images/Vector.svg"></li>
+<li><img src="./Images/Linkedin icon.svg"></li>
+<li><img src="./Images/Vector (2).svg"></li>
+<li><img src="./Images/Vector (1).svg"></li>
+<li><img src="./Images/Vector (3).svg"></li>
+</ul>
```

Comment:
- [ ] Great job so far @AhmedMh0 There are just a few changes you need to make and you'll be on your way to the next project :wink: 
- [ ] I noticed here that all your icon links were broken as a result they are not showing on the page so kindly fix them. :+1: 
- [ ] Aside from that, the icons are on top of the email in the Figma design but this isn't how yours is so kindly fix that as well :+1: 
- [ ] Lastly, please make some style adjustments to your design so it looks similar to the Figma Design :100: 

| Your Design | The Figma Design |
|--------|--------|
|![image](https://github.com/AhmedMh0/Mobile-Portfolio/assets/102699958/c4431f14-4fd3-40d2-bc55-151e1b1d3843) | ![image](https://github.com/AhmedMh0/Mobile-Portfolio/assets/102699958/13083533-9409-4779-b2ad-fd8ac42fe6b7) |

Reply:
![Screenshot 2023-06-14 144749](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/5f481100-132a-4a00-8984-885e802678a8)
your image of figma is wrong and my icons in my browser is seen!
![Screenshot 2023-06-14 145151](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/8f42af5f-4c72-4f5a-b618-2ffc8eb336a0)

Reply:
![Screenshot 2023-06-14 144749](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/5f481100-132a-4a00-8984-885e802678a8)
your image of figma is wrong and my icons in my browser is seen!
![Screenshot 2023-06-14 145151](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/8f42af5f-4c72-4f5a-b618-2ffc8eb336a0)

Reply:
![Screenshot 2023-06-14 144749](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/5f481100-132a-4a00-8984-885e802678a8)
your image of figma is wrong and my icons in my browser is seen!
![Screenshot 2023-06-14 145151](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/8f42af5f-4c72-4f5a-b618-2ffc8eb336a0)

Reply:
Hello, @AhmedMh0 If that is the design you have in your Figma it's fine you can leave it like that. However, please ensure that the image links are working as expected and that they are styled as seen :+1: 
- I noticed that your images directory is spelled with a small letter `i` but in your code, you used capital `I` for specifying the image paths I think that's why the links are broken so kindly fix that :+1: